bgpd/bgp_attr.h

```diff
@@ -160,6 +160,9 @@ struct bgp_attr_srv6_l3vpn {
 };
 
 struct attr_extra {
+	/* PMSI tunnel type (RFC 6514). */
```

Comment:
What is the reason of moving pmsi_tnl_type from attr to attr_extra

Reply:
To save memory, this is the same as with ipv6_ecommunity, ecommunity (in progress), evpn stuff (in progress), etc.